packages/flutter/lib/src/widgets/_platform_selectable_region_context_menu_web.dart

```diff
@@ -106,26 +107,26 @@ class PlatformSelectableRegionContextMenu extends StatelessWidget {
 
   static String _registerWebSelectionCallback(_WebSelectionCallBack callback) {
     _registerViewFactory(_viewType, (int viewId) {
-      final DomHTMLElement htmlElement = createDomHTMLDivElement();
+      final web.HTMLElement htmlElement = web.HTMLDivElement();
```

Comment:
@joshualitt this is throwing an "Illegal Constructor" error. Even if I do this directly in Chrome DevTools I get the same "Illegal Constructor" error. Should we remove this constructor from `package:web` if it's not supposed to be used?

In the meantime, I'm changing this back to `document.createElement('div')`.